package.json

```diff
@@ -69,6 +70,7 @@
     "jest": "^25.3.0",
     "node-sass": "^4.13.1",
     "ts-jest": "^25.3.1",
-    "typescript": "^3.8.3"
+    "typescript": "^3.8.3",
+    "monaco-editor": "^0.20.0"
```

Comment:
I _think_ you'll need to move `monaco-editor` to `dependencies` because we use that in the code.